molecule/test/functional/conftest.py

```diff
@@ -40,6 +40,21 @@
 IS_TRAVIS = os.getenv('TRAVIS') and os.getenv('CI')
 
 
+@pytest.fixture(scope="session", autouse=True)
+def require_installed_module():
+    # import importlib.util
+    # spec = importlib.util.find_spec("molecule")
+    # __requires__ = ["molecule2"]
+    import pkg_resources
+
+    try:
+        pkg_resources.require("molecule")
```

Comment:
why not just "import molecule"?

Reply:
Leftovers, removing them.

Reply:
`import molecule` would succeed if you run tests from repo-root while some tests will fail too. Doing an import tests if module can be imported which is not the same thing as having a package installed. Hint: query version is one of the things failing.